packages/destination-actions/src/destinations/outfunnel/groupIdentifyContact/index.ts

```diff
@@ -0,0 +1,85 @@
+import { ActionDefinition } from '@segment/actions-core';
+import type { Settings } from '../generated-types';
+import type { Payload } from './generated-types';
+
+const getEndpoint = (userId: string) => `https://sink.outfunnel.com/events/segment/${encodeURIComponent(userId)}`;
```

Comment:
Might we need to have the hostname as a separate constant with ability to override it if needed, like for testing?

Reply:
Moved to utils module